set-gpios

```diff
@@ -14,11 +14,23 @@ GPIO_SYSFS=/sys/class/gpio
 
 for i in $(seq ${GPIO_BASE} $((${GPIO_BASE} + ${GPIO_NR})))
 do
-    echo ${i} > ${GPIO_SYSFS}/export
+    if ! echo ${i} > ${GPIO_SYSFS}/export
+    then
+	echo Failed to export gpio${i}
```

Comment:
If you're getting fancy you could test for the directory and if it exists, not attempt to re-export.